src/components/quickReplyModal/quickReplyModalView.tsx

```diff
@@ -0,0 +1,228 @@
+      sheetModalRef.current?.setModalVisible(true);
+      // wait  for modal to open and then show the keyboard
+      setTimeout(() => {
+        inputRef.current?.focus();
+      }, 1000);
+    },
+  }));
+
+  // handlers
+
+  // navigate to post on summary press
+  const _handleOnSummaryPress = () => {
+    Keyboard.dismiss();
+    sheetModalRef.current?.setModalVisible(false);
+    navigate({
+      routeName: ROUTES.SCREENS.POST,
+      params: {
+        content: selectedPost,
+      },
+      key: get(selectedPost, 'permlink'),
+    });
+  };
+
+  // handle submit reply
+  const _submitReply = async () => {
+    let stateTimer;
+    if (isSending) {
+      return;
+    }
+
+    if (currentAccount) {
+      setIsSending(true);
+
+      const permlink = generateReplyPermlink(selectedPost.author);
+
+      const parentAuthor = selectedPost.author;
+      const parentPermlink = selectedPost.permlink;
+      const parentTags = selectedPost.json_metadata.tags;
+      console.log(
+        currentAccount,
+        pinCode,
+        parentAuthor,
+        parentPermlink,
+        permlink,
+        commentValue,
+        parentTags,
+      );
+
+      const status = await postComment(
+        currentAccount,
+        pinCode,
+        parentAuthor,
+        parentPermlink,
+        permlink,
+        commentValue,
+        parentTags,
+      )
+        .then(() => {
+          AsyncStorage.setItem('temp-reply', '');
+
+          stateTimer = setTimeout(() => {
+            setIsSending(false);
+            sheetModalRef.current?.setModalVisible(false);
+            dispatch(
+              toastNotification(
+                intl.formatMessage({
+                  id: 'alert.success',
+                }),
+              ),
+            );
+            clearTimeout(stateTimer);
+          }, 3000);
+        })
+        .catch((error) => {
+          console.log(error);
+          Alert.alert(
+            intl.formatMessage({
+              id: 'alert.fail',
+            }),
+            error.message || JSON.stringify(error),
+          );
+          stateTimer = setTimeout(() => {
+            setIsSending(false);
+            clearTimeout(stateTimer);
+          }, 500);
+        });
+      console.log('status : ', status);
+    }
+  };
+
+  //VIEW_RENDERERS
+
+  const _renderSummary = () => (
+    <TouchableOpacity onPress={() => _handleOnSummaryPress()}>
+      <SummaryArea style={styles.summaryStyle} summary={selectedPost.summary} />
+    </TouchableOpacity>
+  );
+  const _renderAvatar = () => (
+    <View style={styles.avatarAndNameContainer}>
+      <UserAvatar noAction username={currentAccount.username} />
+      <View style={styles.nameContainer}>
+        <Text style={styles.name}>{`@${currentAccount.username}`}</Text>
+      </View>
+    </View>
+  );
+
+  const _renderContent = () => {
+    return (
+      <View style={styles.modalContainer}>
+        {_renderSummary()}
+        {isLoggedIn && _renderAvatar()}
```

Comment:
Not sure if this does any good for user experience, do check the screenshot shared in conversation thread of this PR for more details.

Reply:
hide modal if not logged in